cirq-core/cirq/devices/grid_qubit.py

```diff
@@ -229,20 +235,22 @@ def from_diagram(diagram: str, dimension: int) -> List['GridQid']:
         ...GHIJKL...
         ....IJKL....
         .....KL.....
+        ```
 
         Args:
             diagram: String representing the qid layout. Each line represents
                 a row. Alphanumeric characters are assigned as qid.
-                Dots ('.'), dashes ('-'), and spaces (' ') are treated as
+                Dots ('.'), dashes ('-'), and spaces are treated as
```

Comment:
Why drop the `(' ')`? I think it's helpful just for clarifying that we mean the "space" character, not whitespace in general.